pyproject.toml

```diff
@@ -12,10 +12,9 @@ authors = [
 description = "Megatron's multi-modal data loader"
 readme = "README.md"
 license = "BSD-3-Clause"
-requires-python = ">=3.8"
+requires-python = ">=3.10"
```

Comment:
Huh, we're at 3.10 now? I thought that was still in discussion? 3.9 for sure, but I don't remember finalizing on 3.10